sigle/src/modules/userCard/UserCard.tsx

```diff
@@ -145,12 +150,12 @@ export const UserCard = ({ address, userFollowing }: UserCardProps) => {
               {isLoadingUsername ? '...' : username}
             </Typography>
           </Link>
-          {user && !isLegacy && !following && (
+          {user && user?.username !== username && !isLegacy && !following && (
```

Comment:
the username check here is hiding the button if you see your profile show up on another users' following/followers or the explore page